.github/workflows/coverage-pr.yml

```diff
@@ -22,8 +26,6 @@ jobs:
         with:
           package-manager: yarn
           output: report-markdown
-      - name: Build
```

Comment:
You need to keep this step, because `ArtiomTr/jest-coverage-report-action` left after itself main branch builder code

Reply:
There is already a `build` step before. You are saying `ArtiomTr/jest-coverage-report-action` action is modifying the `dist` folder, and we need to run `build` again?

Reply:
Yes, it chekouts to the main branch, builds it and checkouts back, so it will be main branch artifacts in dist filder

Reply:
Done. Could you give your review again please, @ilfa?